tests/test-servers/express_5_scaffold_server.js

```diff
@@ -0,0 +1,89 @@
+/**
+ * Copyright 2016 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+var has = require('lodash.has');
+var niv = require('npm-install-version');
+niv.install('express@5.0.0-alpha.2');
+var express = niv.require('express@5.0.0-alpha.2');
+var app = express();
+var errorHandler = require('../../index.js')({
+  ignoreEnvironmentCheck: true
+});
+var bodyParser = require('body-parser');
+
+app.use(bodyParser.json());
+
+app.post(
+  '/testErrorHandling'
```

Comment:
actually all of the other files in the directory don't follow the style either, so can be addressed in a follow-on PR.